app/Http/Controllers/AirlineController.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Airline;
+use App\Models\City;
+use Illuminate\Http\JsonResponse;
+use Illuminate\Http\Request;
+use Illuminate\Validation\Rule;
+use Illuminate\View\View;
+
+class AirlineController extends Controller
+{
+    public function index() : View{
+        return view('airlines.show',[
+            'links' => Airline::paginate()->links(),
+            'cities' => City::all()
+        ]);
+    }
+
+    public function store(Request $request) : JsonResponse{
+        $attributes = $request->validate([
+            'name' => ['required', 'unique:airlines'],
+            'description' => ['nullable'],
+            'cities' => ['nullable']
+        ]);
+        $airline = Airline::create([
```

Comment:
In this case I did it "manually" because $attributes[cities] is not part of the airline model itself. I use it in the pivot table airlines_cities. Basically I wanted to create airline model with all attributes except $attributes['cities']